lighthouse-core/gather/gatherers/offline.js

```diff
@@ -21,14 +21,13 @@ class Offline extends Gatherer {
    * @param {LH.Gatherer.LoadData} loadData
    * @return {Promise<LH.Artifacts['Offline']>}
    */
-  afterPass(passContext, loadData) {
+  async afterPass(passContext, loadData) {
```

Comment:
I believe this technically doesn't need to be `async`, though maybe it would be best if it always was :)